### PR TITLE
Set dir and lang on fireplace pages (bug 1155826)

### PR DESCRIPTION
**Pull request by mstriemer**
It makes the spinner go in the right direction.

rtl-from-zamboni mov

### CODE REVIEW

File: mkt/commonplace/templates/commonplace/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html{% if appcache %} manifest="{{ url('commonplace.appcache')|urlparams(repo=repo) }}"{% endif %}>
+<html{% if appcache %} manifest="{{ url('commonplace.appcache')|urlparams(repo=repo) }}"{% endif %}{% if repo == 'fireplace' %} lang="{{ LANG }}" dir="{{ DIR }}"{% endif %}>
```
Comment (eviljeff) [inline mkt/commonplace/templates/commonplace/index.html]: do bad things happen if fireplace isn't special-cased?

Comment (mstriemer) [inline mkt/commonplace/templates/commonplace/index.html]: I don't think so, I just don't think the other repos have the same list of available locales and I'd rather we not set this and then have it get changed once the page loads.

**Comment by mstriemer** [comment]
Added some tests.

**Comment by mstriemer** [comment]
I updated this to always set lang and dir regardless of the app in https://github.com/mozilla/zamboni/commit/7bfc43d422edc0061484ec924e9727d0a274b55c.

**Pull request merged**